### PR TITLE
Add Interactions section to readme

### PR DESCRIPTION
**Pull request by mrcarlosdev**
Provide a way to report bugs and feature requests.

### CODE REVIEW

**Review by DeagleGross — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -66,6 +66,14 @@ This project is licensed under [the MIT License](LICENSE)
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information, see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+## Interactions
+
+You can report bugs or submit feature requests in the [repository's Issues section](https://github.com/Azure/azure-api-management-devops-resource-kit/issues).
+
+Azure API Management DevOps resourceKit has limited "assited support". You can also use [Stack Overflow](https://stackoverflow.com/questions/tagged/azure-api-management-devops-resource-kit) with the `azure-api-management-devops-resource-kit` tag.
```
Comment (tomkerkhove) [inline README.md:73]: Some nits:
```suggestion
Azure API Management DevOps resource kit has limited "assited support". You can use [GitHub Discussions](https://github.com/Azure/azure-api-management-devops-resource-kit/discussions/new) or [Stack Overflow](https://stackoverflow.com/questions/tagged/azure-api-management-devops-resource-kit) with the `azure-api-management-devops-resource-kit` tag.
```

I think it's good to keep the context as close to the repo as we can.

You might want to rephrase "assited support" - What does that mean? Can customers still open support tickets?

File: README.md [hunk 2]
```diff
@@ -66,6 +66,14 @@ This project is licensed under [the MIT License](LICENSE)
...
+
+We welcome and appreciate community contributions. Refer to the [contribution guidelines](https://github.com/Azure/azure-api-management-devops-resource-kit#contributing) for more information.
```
Comment (tomkerkhove) [inline README.md:75]: This is not a contribution guide, we should update the README to point to https://github.com/Azure/azure-api-management-devops-resource-kit/blob/main/CONTRIBUTING.md for detailed info

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mrcarlosdev) [reply README.md]: Added

**Review by tomkerkhove — APPROVED** [review]

**Comment by tomkerkhove** [comment]
@mrcarlosdev I forgot about my comment here:

> You might want to rephrase "assited support" - What does that mean? Can customers still open support tickets?

Can you elaborate on what that means please?

File: README.md [hunk 3]
```diff
@@ -66,6 +66,14 @@ This project is licensed under [the MIT License](LICENSE)
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information, see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+## Interactions
```
Comment (adrianhall) [inline README.md:69]: Change to "Reporting bugs and feature requests" please.

File: README.md [hunk 4]
```diff
@@ -66,6 +66,14 @@ This project is licensed under [the MIT License](LICENSE)
...
+
+You can report bugs or submit feature requests in the [repository's Issues section](https://github.com/Azure/azure-api-management-devops-resource-kit/issues).
+
+Azure API Management DevOps resource kit has limited "assited support". You can use [GitHub Discussions](https://github.com/Azure/azure-api-management-devops-resource-kit/discussions/new) or [Stack Overflow](https://stackoverflow.com/questions/tagged/azure-api-management-devops-resource-kit) with the `azure-api-management-devops-resource-kit` tag.
```
Comment (adrianhall) [inline README.md:73]: "DevOps Resource Kit" - needs proper capitalization.

File: README.md [hunk 5]
```diff
@@ -66,6 +66,14 @@ This project is licensed under [the MIT License](LICENSE)
...
+
+We welcome and appreciate community contributions. Refer to the [contribution guidelines](https://github.com/Azure/azure-api-management-devops-resource-kit/blob/main/CONTRIBUTING.md) for more information.
```
Comment (adrianhall) [inline README.md:75]: Remove this line - it's right above the contributing section.

**Pull request merged**